amundsen_application/static/js/components/common/ListItem/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface SearchIndexParams {
```

Comment:
nit: since these values are used for our logging on the backend at some point we can perhaps rename this to `LoggingParams`, sort of to help decouple concepts/names in the `ListItem` with search since they will be used for things other than search.

Reply:
Hm...and to follow that up, the `ListItem` maybe be used for things other that search, but the logging params might not be. Keeping search in the name would be appropriate then.